### DISCUSSION TITLE
开放源代码定义之历史篇 - 「开源之道」

### DISCUSSION THREAD

**Discussion by giscus** [OCselected/ocselected.github.io#118] [Announcements]
# 开放源代码定义之历史篇 - 「开源之道」

https://opensourceway.community/posts/opensource_culture/the_open_source_definition_history/

**Comment by liuxiyong001** [comment]
所以我提出为”开放源代码”（Open Source)主持商标，以便我们可以将其用于定义。         应该为 注册商标

**Comment by liuxiyong001** [comment]
开源定义来自DeBian指南。开源世界的规则都是实践驱动、自下而上的，连定义也不例外。

**Reply by lijiangsheng1** [reply]
yes
